reactor-core/src/main/java/reactor/core/publisher/FluxMap.java

```diff
@@ -45,7 +63,7 @@
 	 *
 	 * @throws NullPointerException if either {@code source} or {@code mapper} is null.
 	 */
-	FluxMap(Flux<? extends T> source,
+	private FluxMap(Flux<? extends T> source,
```

Comment:
I remember that we prefer internal visibility, but this one is helpful because it forces us to create it through the static factory, and, unless needed, would be better to keep the constructor private